cw-orch-daemon/src/channel.rs

```diff
@@ -2,7 +2,8 @@ use cosmrs::proto::cosmos::base::tendermint::v1beta1::{
     service_client::ServiceClient, GetNodeInfoRequest,
 };
 use cw_orch_core::{environment::ChainInfoOwned, log::connectivity_target};
-use tonic::transport::{Channel, ClientTlsConfig};
+use http::Uri;
```

Comment:
Have you tried this new code with `http://` prefixed URIs ?

Reply:
Yes, see the tests in `query_only.rs`. There's a http and https test there.